Documentation/cli-installation-scenarios.md

```diff
@@ -0,0 +1,237 @@
+2. VS Code 
+
+## Getting Started page
+The page can be found on https://aka.ms/dotnetcoregs.This is the main curated first-run experience for the dotnet CLI. The intent of the page is to help users "kick the tires" quickly and become familiar with what the platform offers. This should be the most stable and curated experience we can offer. The Getting Started page should only point users to curated install experiences that can contain only stable or LKG bits. 
+ 
+
+The below table shows the pertinent information for installs on the "Getting started" page. 
+ 
+| Property              	| Description                                                  	|
+|-----------------------	|--------------------------------------------------------------	|
+| Debian feed           	| Development                                                  	|
+| Local install scripts 	| Latest from rel/1.0.0                                        	|
+
+## Repo landing page
+The repo landing page can be found on: https://github.com/dotnet/cli/readme.md. Download links on the landing page should be decreased in importance. First thing for "you want to get started" section should link to the getting started page on the marketing site. The Repo Landing Page should be used primarily by contributors to the CLI. There should be a separate page that has instructions on how to install both the latest stable as well as latest development with proper warnings around it. The separate page is to really avoid the situation from people accidentally installing unstable bits (since search engines can drop them in the repo first). 
+
+The below table shows the pertinent information for installs on the repo landing page.  
+
+| Property             	| Description                                      	|
+|----------------------	|--------------------------------------------------	|
+| Installation targets 	| Native installers; scripts for curl installs   	|
+| Source branch        	| rel/1.0.0                                        	|
+| Linked builds        	| Latest green build of rel/1.0.0                  	|
+
+# Acquisition modes
+There are multiple acquisition modes that the CLI will have:
+
+1. Native installers
+2. Local installs manually
+3. NuGet packages (for use in other people's commands/code)
+4. Install scripts
+5. Docker
+
+All of these are explained below.
+
+## Native installers
+These installation experiences are the primary way new users are getting the bits. They are aimed towards users kicking the tires. They are found using (not not limited to) the following means:
+
+* Web searches
+* Marketing materials
+* Presentations
+* Documentation
+
+The primary way to get information about this mode of installation is the marketing website. The native installers are considered to be stable by default; this does not imply lack of bugs, but it does imply predictable behavior. They are generated from the stable branches and are never used to get the "bleeding edge" bits. 
```

Comment:
I'm not sure about the term 'marketing web site'. This is going to be a developer-focused landing page.